src/D2L.Security.OAuth2/Validation/Request/IRequestAuthenticator.cs

```diff
@@ -1,10 +1,5 @@
-using System.Net.Http;
+﻿using System.Net.Http;
 using System.Threading.Tasks;
-
-#if !DNXCORE50
-using System.Web;
-#endif

```

Comment:
DNXCORE50 isn't a thing anymore, it was never actually true.
Instead of going with this `#if`-based approach I went with msbuild `Condition` properties/separate files, see above.